src/components/Wearables/SelectorView.tsx

```diff
@@ -31,15 +32,22 @@ export const SelectorView: FC<SelectorViewProps> = (props) => {
           key={item.id}
           onSelected={onSelected}
           selected={item.selected}
-          styles={styles.selectorRow}
+          styles={styles.selectorRow as SelectorRowStyles}
           title={item.title}
         />
       ))}
     </View>
   );
 };
 
-export const SelectorViewDefaultStyles = {
+const defaultStyles = createStyles('SelectorView', () => ({
```

Comment:
I wonder if we'll someday need namespacing of some sort on these style names.  Many classes in this lib are pretty generically named.